python/taichi/lang/ad_fwd_mode_manager.py

```diff
@@ -0,0 +1,30 @@
+from taichi.lang.enums import AutodiffMode
+
+
+class ForwardModeManagerImpl:
```

Comment:
Ideally this context manager should live in python/taichi/ad.py and it should be the user facing api instead of ti.fwdAD?

Reply:
Let me put it in ad.py